README-custom.md

```diff
@@ -0,0 +1,18 @@
+## Using coreos-assembler to build custom FCOS derivatives
+
+The primary goal of Fedora CoreOS is to be useful directly, and
+to have users deploy containers for applications, etc.
+
+Some people and organizations have asked about creating custom
+derivatives of Fedora CoreOS using this project.  You can do so,
+and we make some effort to keep the interfaces between coreos-assembler,
+[fedora-coreos-config](https://github.com/coreos/fedora-coreos-config) etc.. stable.
+
+Since the same tooling is used to build RHEL CoreOS, a derivative of FCOS used as the base of OpenShift 4, being able to build derivatives is a strong requirement.
```

Comment:
Minor: looks like this is missing wrapping.

FWIW, I would reword this as e.g.

> Note that coreos-assembler is used to build both FCOS and RHCOS, so having some level of decoupling is a strong requirement.

It feels a bit odd calling RHCOS a derivative in this context. I mean, yes it's a derivative in the sense that it's a downstream of FCOS, but in terms of tooling in cosa it's definitely on equal footing to FCOS support. And e.g. cosa has lots of code which today is only used for RHCOS. So framing it as a derivative here can imply a stability promise about all derivatives that's stronger than we might want to convey.

Reply:
OK, I dropped this.